mysql-test/suite/clone/include/clone_connection_begin.inc

```diff
@@ -0,0 +1,133 @@
+## Create connection to clone instance optionally restarting instance with
+## monitoring process.
+
+# These variables can to be set before sourcing this file. Currently we used
+# to test both with and without monitoring process.
+#
+# 1. Need to restart mysqld with monitoring process
+# --let inst_monitor = 1
+#
+# 2. Mysqld server instance number for clone
+# --let clone_inst_number = <instance number> 1/2/3 ...
+#     - Instances must be configured in .cnf
+#     - SERVER_PORT_[n] ENV must be set to server PORT in .cnf
+#
+# 3. Mysqld user name for connecting to the instance.
+# --let clone_user = <user_name>
+#
+# 4. Number of connections to be created to the instance
```

Comment:
All of this seems to be unrelated to local clone functionality. Do we need this file at all?